src/styles/globals.css

```diff
@@ -1,3 +1,9 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer base {
+  input[type='search']::-webkit-search-cancel-button {
```

Comment:
This line was needed to remove input type "search"'s unnecessary cancel button

Reply:
I asked ChatGPT about this, and it suggested the following code:
```css
input[type='search']::-webkit-search-cancel-button {
    -webkit-appearance: none;
    appearance: none;
}
```

And I asked the difference between using `appearance: none` and `display: none`, and got the answer below:
> `appearance: none`: This property is used to remove default styling set by the browser. When applied to the `::-webkit-search-cancel-button` pseudo-element, it removes the default styling of the clear button in WebKit-based browsers (like Chrome and Safari). This is often the preferred method for removing the clear button because it specifically targets the element's default appearance without affecting its layout or functionality.

> `display: none`: This property removes an element from the document flow, meaning the element will not occupy any space in the layout. It effectively makes the element and its content invisible and non-interactive. While `display: none` can hide the clear button, it's a more forceful approach. It not only hides the button but also removes its space, which could potentially affect the layout or behavior of the input field.

Reply:
Thank you for searching it!  Indeed, using `apperance : none` seems better! I'll fix it